### PR TITLE
chore: add uwsgi exporter

### PR DESCRIPTION
**Pull request by nitzanmorr**

### CODE REVIEW

File: helm/templates/uwsgi-exporter-container.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+              port: "uwsgimetrics"
+            initialDelaySeconds: 2
+            periodSeconds: 10
+            timeoutSeconds: 5
+            failureThreshold: 3
+            successThreshold: 1
+          ports:
+            - name: "uwsgimetrics"
+              containerPort: {{ .Values.uwsgiExporter.port }}
+{{- end }}
```
Comment (asafMasa) [inline helm/templates/uwsgi-exporter-container.yaml:33]: end of file

File: helm/values.yaml [hunk 2]
```diff
@@ -279,6 +288,24 @@ mapproxinator:
     logPrettyPrintEnabled: false
     responseCompressionEnabled: true
     requestPayloadLimit: 1mb
+  resources:
+    enabled: true
```
Comment (asafMasa) [inline helm/values.yaml:292]: please change to false

**Review by asafMasa — CHANGES_REQUESTED** [review]
good job 👍

File: helm/values.yaml [hunk 3]
```diff
@@ -279,6 +288,24 @@ mapproxinator:
...
+    value:
+      limits:
+        cpu: 500m
+        memory: 500Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+
+uwsgiExporter:
+  enabled: true
```
Comment (CL-SHLOMIKONCHA) [inline helm/values.yaml:302]: consider remove if it is not in use

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]

File: helm/values.yaml (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply helm/values.yaml:302]: can confuse with 'metric.enabled'

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

**Review by asafMasa — APPROVED** [review]

**Pull request merged**